### PR TITLE
Updated sdk version

### PR DESCRIPTION
**Pull request by cemremengu**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/NancyFx/Nancy/pulls) open
- [x] I have verified that I am following the Nancy [code style guidelines](https://github.com/NancyFx/Nancy/blob/45238076ad0b7f6ecabd6bae8469e30458d02efe/CONTRIBUTING.md#style-guidelines)
- [x] I have provided test coverage for my change (where applicable)

### CODE REVIEW

**Comment by khellang** [comment]
The build scripts needs to be updated as well :smile: https://github.com/NancyFx/Nancy/blob/master/build.sh#L31-L32

**Comment by thecodejunkie** [comment]
So would the `global.json` in the other repos (bootstrappers etc)

File: build.ps1 [hunk 1]
```diff
@@ -54,7 +54,7 @@ Function Install-Dotnet()
 
     # Run the dotnet CLI install
     Write-Output "Installing Dotnet CLI version 1.0.0-preview2-003121..."
```
Comment (jchannon) [inline build.ps1]: Can you change this comment to `3131`. Thanks

**Comment by cemremengu** [comment]
@jchannon I saw that #2594  was merged. If this PR is dead I will close it

**Comment by jchannon** [comment]
Apologies #2594 updated to latest sdk but also changed the build script quite a bit ❤️

**Comment by jchannon** [comment]
Apologies. 2594 changes the whole build script as well as add latest Sdk

**Pull request closed without merging**